src/plugins/intel_cpu/src/nodes/kernels/jit_kernel_base.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <math.h>
+#include <dnnl_extension_utils.h>
+#include <cpu/x64/jit_generator.hpp>
+
+namespace ov {
+namespace intel_cpu {
+
+using namespace dnnl::impl;
+namespace x64 = dnnl::impl::cpu::x64;
+
+template<typename jit_params, typename jit_call_args>
+class jit_kernel_base : public x64::jit_generator {
```

Comment:
Please rebase other kernels : gather and rdft on this. It can highlight potential issues here.

Reply:
Done